courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<ul><li>Before this cell add a title like "AutoML Vision Setup"</li><li>"a Project on GCP Console " -&gt; "a project on the GCP Console"</li><li>For links in markdown, use the construct </li></ul><pre class="ql-syntax" spellcheck="false">[link text](link url)
</pre><ul><li>Instead of referring to the documentation, maybe add a bash cell with the command to enable the API programmatically</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>done</p>